RHDEVDOCS-5954: Fixing typos in pre-reqs (#74766)
openshift/openshift-docs: accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.adoc

[id="prerequisites_configuring-sso-for-argo-cd-using-keycloak"]
== Prerequisites
* Red Hat SSO is installed on the cluster.

> * You have installed the {gitops-title} Operator on your {OCP} cluster.

Reviewer: This phrasing makes the middle item no longer parallel construction to the others. Does this work?
```suggestion
* The {gitops-title} Operator is installed on your {OCP} cluster.
```

Author: Fixed.
